include/libtorrent/alert_types.hpp

```diff
@@ -2064,6 +2065,8 @@ namespace libtorrent
 		boost::array<char, 64> signature;
 		std::string salt;
 		boost::uint64_t seq;
+
+		bool success;
```

Comment:
could you please add a comment describing what this means? (if this is false, does it mean it definitely failed, or may there be another put that succeeds later?)